AutoRest/TestServer/swagger/httpInfrastructure.json

```diff
@@ -2795,6 +2795,9 @@
       }
     },
     "A": {
+      "x-ms-exception": {
```

Comment:
In that case can you make it a general purpose extension: `x-ms-client-name: name` which can be applied to any model (not just errors). The less extensions the better.